packages/message-manager/src/utils.test.ts

```diff
@@ -263,6 +263,18 @@ describe('utils', () => {
         ),
       ).not.toThrow();
     });
+
+    it('should not throw if data is correct (object format)', () => {
```

Comment:
Perhaps since we now accept two formats, is it worth modifying the test above this to say "should not throw if data is correct (JSON-parseable format)" or "should not throw if data is correct (string format)"?

Reply:
I think this test is perfectly fine as it is, since the behaviour hasn't changed, only difference is that now we accept both objects & JSON strings. This test is to ensure that passing in an object is acceptable.